DigitalLearningSolutions.Web/Views/Support/Faqs/_SearchableFaqCard.cshtml

```diff
@@ -0,0 +1,37 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Support.Faqs
+@model SearchableFaqViewModel
+
+@{
+  var viewFaqRouteData = new Dictionary<string, string?> {
+    { "dlsSubApplication", Model.DlsSubApplication.UrlSegment },
+    { "faqId", Model.Faq.FaqId.ToString() },
+  };
+}
+
+<div class="searchable-element nhsuk-expander-group word-break" id="@Model.Faq.FaqId-card">
+
+  <details class="nhsuk-details nhsuk-expander">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text" id="@Model.Faq.FaqId-name">
+        <span class="searchable-element-title" name="name">@Model.Faq.QText (@Model.Faq.FaqId / @Model.Faq.Weighting)</span>

```

Comment:
Just spotted the title ends with "({Id}/{Weighting})", guessing this was a debugging helper that was missed.

Reply:
Aha, yes. Good spot. Sorted.